.circleci/config.yml

```diff
@@ -37,11 +51,36 @@ jobs:
             - vendor
             - resources
             - app
+      - run:
+          name: Create implementation code by extending BaseIndex
+          command: |
+            cat <<'PHPSRC' > /var/www/mysite/www/app/src/TestIndex.php
+            <?php
+              use Firesphere\SolrSearch\Indexes\BaseIndex;
+              use SilverStripe\CMS\Model\SiteTree;
+              use SilverStripe\Dev\TestOnly;
+              class TestIndex extends BaseIndex implements TestOnly {
+              public function init() {
+                $this->addClass(SiteTree::class);
+                $this->addFulltextField('Title');
+                $this->addFulltextField('Content');
+              }
+              public function getIndexName() {
+                return 'TestIndex';
+              }
+            }
+            PHPSRC
       - run: vendor/bin/sake dev/build
       #      No tests to run yet
       #      - run: vendor/bin/phpunit --coverage-clover=coverage.xml --log-junit ./coverage/junit.xml --coverage-html ./coverage/ -d memory_limit=512M tests/unit
 
       - run: vendor/bin/phpcs --standard=phpcs.xml.dist src
+      - run:
+          name: Run Solr Configure
+          command: |
+            mkdir -p /var/www/mysite/www/.solr && chmod -R 777 /var/www/mysite/www/.solr
+            vendor/bin/sake dev/tasks/SolrConfigureTask
+      - run: vendor/bin/sake dev/tasks/SolrIndexTask
```

Comment:
Please run these _before_ the unit tests (after rebasing), so we can actually use it for testing.

You rock! 🎈